pvlib/pvsystem.py

```diff
@@ -164,15 +165,16 @@ class PVSystem:
 
     inverter : string, optional
         The model name of the inverters.
-        May be used to look up the inverter_parameters dictionary
+        May be used to look up the ``inverter_parameters`` dictionary
```

Comment:
Since we're editing this docstring, I would delete "May be used....via some other method." pvlib doesn't use `PVsystem.inverter` for anything, it is here to let a user attach a human-readable name (string) for convenience.

Reply:
ok --- deleted these lines from `module` and `inverter` in `PVSystem`